### ISSUE TITLE
le fichier RSS est vide  (backend)

### ISSUE THREAD

**Issue by touti** [QuartiersXXI/quartiersxxi#16]
le fichier est vide http://quartiersxxi.org/spip.php?page=backend

**Comment by rastapopougros** [comment]
En fait il n'est pas vide (faire Ctrl + U pour voir la source), mais si Firefox n'affiche rien ça veut dire qu'il y a des erreurs de validation quelque part.

http://validator.w3.org/feed/check.cgi?url=http%3A%2F%2Fquartiersxxi.org%2Fspip.php%3Fpage%3Dbackend

Le mieux serait peut-être de passer tout ça en Atom, mais bon dans tous les cas ya des choses à corriger.

**Comment by touti** [comment]
Effectivement, bizarre, pourquoi cette image png  bascule-t-elle sur une base64 en passant par le fichier rss ?
